TideApp/TideApp/Helpers/CLGeocoder+LocationName.swift

```diff
@@ -35,3 +33,9 @@ extension CLGeocoder {
     }.eraseToAnyPublisher()
   }
 }
+
+fileprivate extension CLPlacemark {
+  func getLocationNameIfPresent() -> String? {
```

Comment:
Minor, but could this be a computed property rather than a function? `var locationName: String?`

Reply:
done